### DISCUSSION TITLE
Quale Shelly...?

### DISCUSSION THREAD

**Discussion by xmxbx73** [indomus/forum#3133] [Hardware & Firmware]
Buongiorno a tutti, 
Mi hanno appena installato dei motori alle persiane, comandati da interruttori su/giù. Sono già automatizzati, nel senso con una pressione breve dell' interruttore si aprono/chiudono fino a fine corsa. Vorrei renderli più smart con degli Shelly. Credo che mi basti un dispositivo con contatti puliti. Che modello di Shelly mi servirebbe secondo voi?
Vi ringrazio anticipatamente per ogni consiglio.
Bye, Marco

**Comment by Scialla** [comment]
non ho tapparelle motorizzate ma, da rapida ricerca, sembra che gli Shelly Plus 2PM siano quelli attualmente in commercio atti allo scopo; prima si usavano gli Shelly 2.5

**Comment by angio89** [comment]
Ciao,
io possiedo tapparelle motorizzate ma non automatizzate come le tue, nel senso che devo tenere premuto per poter alzare/abbassare la cover. Ho collegato, come già detto da Scialla, degli Shelly Plus 2PM e funziona tutto alla grande. Per il discorso degli input fisici diversi credo basti una configurazione diversa dello Shelly che dovrà dare solo un singolo impulso anzichè rimanere alimentato fino al fine corsa. Tra l'altro, i fine corsa sono meccanici?

**Comment by xmxbx73** [comment]
Ciao, 
Grazie per le risposte.
Ho già utilizzato gli Shelly Plus 2PM su 2 oscuranti 'non automatizzati', ma mi sembravano un po' sprecati per utilizzarli come semplici pulsanti contatti puliti .
😉
